todo-cli.cabal

```diff
@@ -0,0 +1,54 @@
+-- This file has been generated from package.yaml by hpack version 0.20.0.
+--
+-- see: https://github.com/sol/hpack
+--
+-- hash: 592e31a8bba4a31e2540e7aea82b3da47430f4c1a8a18f898de13cd3be53b9a9
+
+name:           todo-cli
+version:        0.1.0.0
+description:    Please see the README on Github at <https://github.com/StasyaGera/todo-cli#readme>
+homepage:       https://github.com/StasyaGera/todo-cli#readme
+bug-reports:    https://github.com/StasyaGera/todo-cli/issues
+author:         Stanislava Gera
+maintainer:     stasya.gera@yandex.ru
+copyright:      2018 Stanislava Gera
+license:        BSD3
+license-file:   LICENSE
+build-type:     Simple
+cabal-version:  >= 1.10
+
+extra-source-files:
+    ChangeLog.md
+    README.md
+
+source-repository head
+  type: git
+  location: https://github.com/StasyaGera/todo-cli
+
+library
+  hs-source-dirs:
```

Comment:
Could you please format all fields here just like in this project?

* https://github.com/serokell/importify/blob/master/importify.cabal

Aligning field names to the left and field values to the right helps to observe values easier.